src/instrument/utils/conversions.py

```diff
@@ -0,0 +1,14 @@
+"""
+Miscellaneous conversion functions.
+
+.. automodule::
+
+    ~ts2iso
+"""
+
+import datetime
+
+
+def ts2iso(ts: float, sep: str = " ") -> str:
+    """Convert Python timestamp (float) to IS8601 time in current time zone."""
+    return datetime.datetime.fromtimestamp(ts).isoformat(sep=sep)
```

Comment:
Any reason why this is now in its own file? Also how often are we using this function that we need it to be it's own thing?

Reply:
Already existed in another file, which also imported from `bluesky`.  Original statement included:

> We would prefer to have a plain python function independent from Bluesky RE